src/app/notes/notes.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, OnInit } from '@angular/core';
+
+
+@Component({
+  selector: 'notes',
+  templateUrl: './notes.component.html',
+  styleUrls: ['./notes.component.css']
+})
+export class NotesComponent implements OnInit {
+
+	/*
+	* Data members for the notes component, title and content of the notes and patient ID
+	* for the corrosponding patient. 
+	*/
+
+	// public title: string;
+	public content: string;
+	public id: string;
+	
+
+  constructor() { 
+  	this.content=''; 
+  	this.id='';
+  }
+
+  public ngOnInit() {
+  }
+
+  // Loads a note from database based on patient ID
+	// for now loads in dummy text.
+	// TODO:Connect client side api
+	public loadNotes() {
+		let loadedNote: Object[] = this.getNotesQuery();
+
+		this.content = loadedNote['content'];
+		this.id = loadedNote['id'];
+	}
+
+	// Placeholder for CLIENT/API getNotes request
+	public getNotesQuery() {
```

Comment:
Unless it's needed for Angular to work, I don't see any reason for these methods or data members to be public.

Reply:
It's needed for angular: Notes will be passed up to the parent component.

Reply:
From what I read on the angular docs it is necessary to make them public.